src/client/app/redux/selectors/maps.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RootState } from 'store';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectMapState = (state: RootState) => state.maps;
```

Comment:
I don't know if getting RootState is what is now normally done so this may go away in the update to Redux usage.

Reply:
I believe this may have been addressed in commit [811dc66](https://github.com/OpenEnergyDashboard/OED/pull/1314/commits/811dc6693629b4bef6f78de78981095a93f66e0d)

Reply:
I'm sorry but I still see state being directly handled. Do you see otherwise?

Reply:
I'm not sure what you mean by state being directly handled. I see the use of createAppSelector() in other selector files, would you like me to use this instead?

Reply:
I have made changes in commit [d270242](https://github.com/OpenEnergyDashboard/OED/pull/1314/commits/d270242dbf65ef6dce23cefc65ce95fb3610b163) that I believe solves this issue.

Reply:
He may be referring to the fact that these selectors are not using the same patterns of accessing data as meters, groups, etc.. Additionally many existing selectors, when possible, are defined  in slices/ exported. There aren't many using 
(state:Root state)=> state.select.my.thing;
The difference here is that maps isn't using RTKQ, or rtk slices, so I think this is fine and will change once the data fetching also migrates over where it can use the entityAdapter pattern like other apiData. 

@huss correct me if I'm mistaken

Reply:
Yes, this is an older way that works but should be updated. It isn't part of the issue linked so it did not have to happen but I wanted the developers to know. I am noting this since some other state changes were made. More and more I'm leaving comments on PRs about items that need to happen (future changes, documentation, etc.)